Library/Formula/bzr-fastimport.rb

```diff
@@ -0,0 +1,54 @@
+class BzrFastimport < Formula
+  desc "Bazaar plugin for fast loading of revision control"
+  homepage "https://launchpad.net/bzr-fastimport"
+  url "https://launchpad.net/bzr-fastimport/trunk/0.13.0/+download/bzr-fastimport-0.13.0.tar.gz"
+  sha256 "5e296dc4ff8e9bf1b6447e81fef41e1217656b43368ee4056a1f024221e009eb"
+
+  depends_on :python if MacOS.version <= :snow_leopard
+  depends_on "bazaar"
+
+  resource "python-fastimport" do
+    url "http://launchpad.net/python-fastimport/trunk/0.9.0/+download/python-fastimport-0.9.0.tar.gz"
+    sha256 "07d1d3800b1cfaa820b62c5a88c40cc7e32be9b14d9c6d3298721f32df8e1dec"
+  end
+
+  def install
+    resource("python-fastimport").stage do
+      system "python", *Language::Python.setup_install_args(libexec/"vendor")
+    end
+
+    libexec.install Dir["*"]
+  end
+
+  def post_install
```

Comment:
Why this need to be put in `post_install`? I mean you can write to `var` directly during `install`